specification/columns/skuid.md

```diff
@@ -2,7 +2,12 @@
 
 A SKU ID is a unique identifier that defines a provider-supported construct for organizing properties that are common across one or more [*SKU Prices*](#glossary:sku-price). SKU ID can be referenced on a catalog or [*price list*](#glossary:price-list) published by a provider to look up detailed information about the SKU. The composition of the properties associated with the SKU ID may differ across providers. Some providers may not support the [*SKU*](#glossary:sku) construct and instead associate all such properties directly with the *SKU Price*. SKU ID is commonly used for analyzing cost based on *SKU*-related properties above the pricing constructs.
 
-The SkuId column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider publishes a SKU list. This column MUST be of type String. It MUST NOT be null when [ChargeClass](#chargeclass) is not "Correction" and [ChargeCategory](#chargecategory) is "Usage" or "Purchase", MUST be null when ChargeCategory is "Tax", and MAY be null for all other combinations of ChargeClass and ChargeCategory. SkuId MUST equal SkuPriceId when a provider does not support an overarching SKU ID construct.
+The SkuId column adheres to the following requirements:
+
+* The SkuId column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider publishes a SKU list.
+* This column MUST be of type String.
+* It MUST NOT be null when [ChargeClass](#chargeclass) is not "Correction" and [ChargeCategory](#chargecategory) is "Usage" or "Purchase", MUST be null when ChargeCategory is "Tax", and MAY be null for all other combinations of ChargeClass and ChargeCategory.
+* SkuId MUST equal SkuPriceId when a provider does not support an overarching SKU ID construct.
```

Comment:
This has 4 different ways to reference the column. Can we pick one? My preference is the last one since it's the simplest.
```suggestion
* SkuId MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider publishes a SKU list.
* SkuId MUST be of type String.
* SkuId MUST NOT be null when [ChargeClass](#chargeclass) is not "Correction" and [ChargeCategory](#chargecategory) is "Usage" or "Purchase".
* SkuId MUST be null when ChargeCategory is "Tax".
* SkuId MAY be null for all other combinations of ChargeClass and ChargeCategory.
* SkuId MUST equal SkuPriceId when a provider does not support an overarching SKU ID construct.
```

Reply:
I'll copy this comment to WorkItem #557, as this is one of the fundamental aspects we plan to address across the board as part of that WorkItem.

> ...
> _Use consistent wording, phrases, and terminology throughout the document_
> ...

On that note, @flanakin  I would appreciate your feedback on the [24.11.26 - Normative Requirements Refinement](https://docs.google.com/document/d/1fKeGCBIFQqx9puEpWdIT9CpGSn_-Gh86ky1bf_5-x5Y/edit?tab=t.0), particularly the Guidelines/Principles and the Template/Cookbook